### PR TITLE
feat(callout): Fix Bug 1462835, replace saucelabs banner with new perception survey

### PR DESCRIPTION
**Pull request by schalkneethling**
Replace saucelabs banner with new perception survey banner. As we will not switch back to the Saucelabs banner, I took the opportunity to make this style banner standard going forward.

Also note, I am waiting on Dan and Matiej with regards to the `utm` parameters. So, good for review but, wait on merge.

### CODE REVIEW

File: kuma/static/styles/components/home/column-callout.scss [hunk 1]
```diff
@@ -239,36 +238,51 @@ $saucelabs-callout-image-spacing: 129px + 20px;
...
+
+.callout-survey {
+    &:after {
+        background-image: url($path-to-embedded-images + 'promos/survey.svg');
+    }
+}
+
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+    .callout-survey a:after {
+        background-image: url($path-to-embedded-images + 'promos/survey.svg');
```
Comment (jwhitlock) [inline kuma/static/styles/components/home/column-callout.scss]: I think I led you down the wrong path. Here's what the banner looks like now:

screenshot 2018-05-29 07 55 45

Maybe we should return to ``callout-standard`` for this rule.

Comment (schalkneethling) [reply kuma/static/styles/components/home/column-callout.scss]: Interesting that you see that, I guess I did not tests again after rebasing. I had a hell of a time rebasing and some thing clearly got lost in translation. I will resolve this issue. Having been without power for the entire day :-/ I will take care of this first thing tomorrow. Thanks for highlighting this.

Comment (jwhitlock) [reply kuma/static/styles/components/home/column-callout.scss]: Sorry about your rough day! If it helps, here's the branch before the latest rebase:

https://github.com/jwhitlock/kuma/tree/survey-banner-1462835

**Comment by schalkneethling** [comment]
@jwhitlock Alrighty, updated.

File: kuma/static/styles/components/home/column-callout.scss (hunk 1 above)
Comment (jwhitlock) [reply kuma/static/styles/components/home/column-callout.scss]: Much better!

screenshot 2018-05-30 08 51 57

**Pull request merged**